rfcs/text/0001-rfc-process.md

```diff
@@ -70,26 +86,66 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs. Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in the DCP Zenhub board.
+
+To inform the DCP Community about the development of an RFC, with the intention of forming collaborations between people thinking about similar problems, it is recommended that a draft of an RFC be first published once the Problem Statement and Authors have been established. One possible process that is recommended is as follows:
+
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
+   5. Tag the PR with the "rfc-draft" label.
+
+#### **Non-Authors**:
+
+When an RFC _Draft_ PR is created and furthermore labeled with "rfc-draft" commentary is not allowed since the RFC is under active development. This time period is specifically an opportunity for collaboration on authorship so if an RFC sparks interest and you would like to collaborate on it (especially if you have been thinking about the problem statement yourself), reach out the Author(s) directly and offline. Specific commentary whether agreement or disagreement on the RFC is discouraged until the Author(s) is/are ready for the PR to be proposed/reviewed.
```

Comment:
The opening and closing sentences read as redundant? How about something like:

When an RFC _Draft_ PR is published and ~furthermore~ labeled as "rfc-draft"**,** specific commentary is discouraged (negative or positive) until the Authors(s) propose the RFC for community review. If the draft RFC sparks interest, reach out to the Author(s) offline to suggest a collaboration.

Reply:
Love it. Thank you!